packages/gafl-webapp-service/src/pages/multibuy/view-licences/view-licences.njk

```diff
@@ -3,13 +3,15 @@
 {% from "button/macro.njk" import govukButton %}
 {% from "summary-list/macro.njk" import govukSummaryList %}
 
+{% set title = mssgs.multibuy_view_licences_title %}
+{% block pageTitle %}{{ title }}{{ mssgs.gov_uk }}{% endblock %}
 
 {% block content %}
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-l">Selected Licences</h1>
+        <h1 class="govuk-heading-l">Selected licences</h1>
```

Comment:
This has been missed when we were moving all hardcoded strings to be labels. Please could you add a new label in en.json and cy.json, then replace the text here with that new label?